src/pages/LendingTeam/LendingTeam.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<www-page>
+		<kv-page-container>
+			<kv-grid class="tw-grid-cols-6
```

Comment:
These class attribute values can likely all fit on one line (120 char limit in `ui`).